### PR TITLE
Standardization

### PR DESCRIPTION
**Pull request by karikarshivani**
### [Report](https://github.com/karikarshivani/InSpecReports/blob/master/Standardization/AWS_RDS_Oracle12c_STIG_Before.json) before changes
---
**Questions for @ejaronne:**
- [x] README - [model](https://github.com/CMSgov/cms-ars-3.1-moderate-red-hat-enterprise-linux-7-stig-overlay/blob/master/README.md)
- [x] in inspec.yml, validate the title and the summary to be consistent
- [x] in inspec.yml, set `inspec_version: ">= 4.0"`
- [x] in inspec.yml, validate if any supports statements are valid at the overlay and baseline levels
- [x] in inspec.yml, set path to URL style
- [x] in inspec.yml, change from attributes to inputs
- [x] in inspec.yml, set `profile: ` for any inputs meant to override the baseline settings
- [x] in inspec.yml, set `sensitive: ` flags as appropriate, e.g., for user names, passwords, hostnames, etc.
  - ~~Please lmk if I should add the flag to any other inputs~~
- [x] in controls/, remove any instances of changes to the NIST tag and replace as needed according to the appropriate approach indicated in the [lifecycle SOP](https://saf.cms.gov/CMS_InSpec_Profile_Lifecycle_SOP_v1.0_20190702.pdf)
- [x] Update the License.md
---
### [Report](https://github.com/karikarshivani/InSpecReports/blob/master/Standardization/AWS_RDS_Oracle12c_STIG_After.json) after changes

### CODE REVIEW

**Comment by karikarshivani** [comment]
Screen Shot 2020-11-13 at 3 59 44 PM

**Comment by karikarshivani** [comment]
@ejaronne Is it not required to configure AWS CLI for this overlay?

**Comment by ejaronne** [comment]
> @ejaronne Is it not required to configure AWS CLI for this overlay?

There are no AWS CLI calls in these database profiles. This overlay simply sets the operating system checks to N/A, and makes certain adjustments within the database queries.

File: inspec.yml [hunk 1]
```diff
@@ -1,135 +1,161 @@
...
+    sensitive: true
+    profile: aws-rds-oracle-database-12c-stig-baseline
 
   - name: host
     description: 'hostname Oracle DB (e.g., ''localhost'')'
     value: ''
+    sensitive: true
+    profile: aws-rds-oracle-database-12c-stig-baseline
 
   - name: service
```
Comment (karikarshivani) [inline inspec.yml:34]: Discussion with @ejaronne : Add sensitive flag

File: inspec.yml [hunk 2]
```diff
@@ -1,135 +1,161 @@
...
     description: 'service name Oracle DB (e.g., ''ORCLCDB'')'
     value: ''
+    profile: aws-rds-oracle-database-12c-stig-baseline
 
   - name: sqlplus_bin
```
Comment (karikarshivani) [inline inspec.yml:40]: Discussion with @ejaronne : Add sensitive flag

File: inspec.yml (hunk 1 above)
Comment (karikarshivani) [reply inspec.yml:34]: 4587cda6919278aa323e8de72b6d8dbb8cf8ec3f

File: inspec.yml (hunk 2 above)
Comment (karikarshivani) [reply inspec.yml:40]: 4587cda6919278aa323e8de72b6d8dbb8cf8ec3f

**Pull request merged**